ODBC.pck.st

```diff
@@ -1602,28 +1602,28 @@ initialize
 	digits := 0.
 	size := 0.! !
 
-!ODBCColumn methodsFor: 'private - type convertion' stamp: 'fap 8/14/2009 13:10'!
+!ODBCColumn methodsFor: 'private - type convertion' stamp: 'RMV 10/30/2024 20:54:58'!
 asNumber: aString 
 	"creates a Number from aString. Could be an Integer or a Fraction"
 	| stream zero sign integerPart char fractionPart scale |
 	stream := aString readStream.
 	""
-	zero := $0 asInteger.
+	zero := 0.
 	sign := (stream peekFor: $-)
 				ifTrue: [-1]
 				ifFalse: [1].
 	""
 	integerPart := 0.
 	[stream atEnd
 		or: [(char := stream next) == $.]]
-		whileFalse: [integerPart := integerPart * 10 + (char asInteger - zero)].
```

Comment:
There is no `asInteger` method in the `Character` class, but we can use the `digitValue` method instead.

Reply:
I haven't reviewed these changes but was just taking a look at the README. For what is worth, beware! `$a asInteger ~= $a digitValue`.